src/components/StatsScoringDialog.vue

```diff
@@ -17,73 +17,66 @@
           variant="text"
           data-cy="ranked-info-button"
         >
-          <span v-if="showButtonText">How are ranks determined?</span>
+          <span v-if="showButtonText">{{ t('game.dialogs.statsScoringDialog.ranksDetermination') }}</span>
           <v-icon class="ml-1" icon="mdi-information-outline" aria-hidden="true" />
         </v-btn>
       </slot>
     </template>
     <!-- Dialog -->
     <template #body>
       <p>
-        Competitive Cuttle is divided into 4 seasons per year, one per suit: Clubs, Diamonds, Hearts, and
-        Spades. At the end of each season, the top 8 players play a double elimination championship tournament
-        and the
-        <strong>champions are permanantly accoladed on the site.</strong>
+        {{ t('game.dialogs.statsScoringDialog.competitiveCuttle.description') }}
+        <strong>{{ t('game.dialogs.statsScoringDialog.competitiveCuttle.highlight') }}</strong>
       </p>
       <div class="d-flex justify-space-around flex-wrap mt-4">
         <AwardCard username="Champion player" :place="1" class="mb-4" />
         <AwardCard username="Second Place Player" :place="2" class="mb-4" />
         <AwardCard username="Third Place Player" :place="3" class="mb-4" />
       </div>
       <p>
-        Each Season is divided into 13 weeks. For each week, we count the number of best 2/3 ranked matches
-        each player wins against unique opponents (ignoring stalemates) and assign players points based on
-        their weekly standing.
+        {{ t('game.dialogs.statsScoringDialog.seasonDescription') }}
       </p>
       <v-list class="mt-4" bg-color="surface-2" base-color="surface-1">
         <v-list-item>
           <v-chip variant="elevated" class="mr-2 mb-1" :color="theme.firstPlace">
-            5 Points
+            5 {{ t('game.dialogs.statsScoringDialog.points.displayName') }}
```

Comment:
Let's refactor the points displayName to input a count and pluralize using the same key
```suggestion
            {{ t('game.dialogs.statsScoringDialog.points.displayName', 5) }}
```